actions/contacts/unconfirm.go

```diff
@@ -41,3 +41,20 @@ func (a *Action) unconfirm(c *gin.Context) {
 	}
 	c.Status(http.StatusOK)
 }
+
+// unconfirmContact will unconfirm contact request
+// Unconfirm contact godoc
+// @Summary		Unconfirm contact
```

Comment:
Same as for accept: use single method adjust description and use // @DeprecatedRouter

Reply:
done (single method adjust description and use // @DeprecatedRouter)